trusted-types/Element-setAttribute-not-enforced-in-non-TT-documents-globals-CSP.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
```

Comment:
This won't work as required. You need to have the element inside the iframe (and have TT enforced in the iframe not the parent document) and then move it to the parent. CSP inherits down into the iframe as currently written so this will fail